cli/bin/semgrep

```diff
@@ -71,10 +78,11 @@ def find_semgrep_core_path(core="semgrep-core", extra_message=""):
     if path is not None:
         return path
  
-    print(f"Failed to find {core} in PATH or in the semgrep package.{extra_message}",
-          file=sys.stderr)
-    # fatal error, see src/osemgrep/core/Exit_code.ml
-    sys.exit(2)
+    raise CoreNotFound(f"Failed to find {core} in PATH or in the semgrep package.{extra_message}")
+
+def exec_legacy_semgrep():
```

Comment:
rename to exec_pysemgrep (that's the name we usually use, and consistency is good).